### PR TITLE
stylus stylesheets

### PR DESCRIPTION
**Pull request by gambhiro**
For #126.

### CODE REVIEW

File: data/styles/epub3-fonts.css [hunk 1]
```diff
@@ -2,93 +2,77 @@
   font-family: "Noto Serif";
   font-style: normal;
   font-weight: normal;
-  src: url(../fonts/notoserif-regular-latin.ttf);
+  src: url("../fonts/notoserif-regular-latin.ttf");
```
Comment (mojavelinux) [inline data/styles/epub3-fonts.css]: My understanding is that the quotes are not required when you use the round bracket enclosure. Is there a reason you are adding the double quotes?

Comment (mojavelinux) [inline data/styles/epub3-fonts.css]: Or is this what Stylus added?

**Comment by mojavelinux** [comment]
The CSS files should be removed. We should only store the Stylus files in the repository. We'll still need to work out when to compile the Stylus files, but it should be handled by the build.

**Comment by mojavelinux** [comment]
I don't think we should put the Stylus files in the data folder. That folder is the data folder for the packaged gem (it's the data for the gem). Instead, we need another location in the project. I'd like to propose one of these two locations:

* assets/styles
* styles

I'm open to other suggestions.

File: data/styles/epub3-fonts.css (hunk 1 above)
Comment (mojavelinux) [inline data/styles/epub3-fonts.css]: Answered my own question. I wonder if this is something we can configure.

**Comment by gambhiro** [comment]
The recent commits include:

I dicovered that stylus has a converter under the `--css` option, and this one preserves the comments and original structure more closely, so I re-converted from the css to stylus.

After that, I added an include file `colors.styl` with the available color hex codes. The color values are then associated by semantic function and these are used in the `epub3.styl` file.

```
$dark-gray = #666665

$link-border = $dark-gray

body a:link
  border-bottom: 1px dashed $link-border
```

I moved the stylus files to assets/styles folder.

My goal in this PR was to get the CSS from stylus files, and that's good now. At least with this much it was clear what to do. I know you mentioned considering how and when the CSS should be generated from stylus, but it's not very clear at the moment and it could be addressed later.

I recommend that this is ready to merge, unless you spot anything to fix.

**Comment by gambhiro** [comment]
Ah yes, and adding quotation marks to the font url regex in `packager.rb`.

**Comment by gambhiro** [comment]
Please mention if there is a problem with this PR, or if you simply haven't had the time to review it.

**Pull request closed without merging**